src/config/config-initializer.js

```diff
@@ -0,0 +1,33 @@
+// LICENSE : MIT
+'use strict';
+const Promise = require('bluebird');
+const fs = require('fs');
+const path = require('path');
+const Config = require('../config/config');
+
+// Public Interface
+/**
+ * Initializer class for config of textlint.
+ */
+const init = {
+    /**
+     * Create .textlintrc file in the current working directory
```

Comment:
I think that *current working directory* should be optional.
It means that `initializeConfig(dir)` accept `dir` option.
It is useful for testing.